fmt/numbers.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
+// A module to format numbers.
+
+/** Add zeros to the beginning of a number till it reaches a certain digit count. */
```

Comment:
```suggestion
/** Add zeros to the beginning of a number until it reaches a certain digit count. */
```

Reply:
removed the entire file because it was pointless